third_party/move/move-compiler-v2/tests/checking/variants/variants_check_err.exp

```diff
@@ -18,6 +18,12 @@ error: variants not allowed in this context
 22 │         0x815::m::missplaced_variant::Red();
    │                                       ^^^
 
+error: the function takes 1 argument but 0 were provided
```

Comment:
Why do we have a new error here?  Get rid of it, please.

Reply:
done